extension/src/paymentHandler/payment-handler.js

```diff
@@ -36,7 +38,16 @@ async function handlePayment (paymentObject) {
   return { success: true, data: handlerResponse }
 }
 
+function _isRefund (paymentObject) {
```

Comment:
we could move this method to utility class

Reply:
Why? It is used only in this class.